packages/templates/golang.yaml

```diff
@@ -0,0 +1,22 @@
+{{ define "golang_deps" }}
+{{ if .Values.distribution }}
+{{if eq .Values.distribution "opensuse" }}
+- zypper in -y git {{- if .Values.upx -}} upx {{- end }}
+{{else if eq .Values.distribution "fedora" }}
+- dnf install -y git {{- if .Values.upx -}} upx {{- end }}
+{{else if eq .Values.distribution "ubuntu" }}
+- apt-get install -y git {{- if .Values.upx -}} upx {{- end }}
+{{end}}
+{{end}}
+{{end}}
+
+{{ define "golang_download_package" }}
+{{ $version:= get . "version" }}
+{{ $org:= get . "org" }}
+{{ $repo:= get . "repo" }}
+- |
```

Comment:
maybe we should add `{{ $host:= get . "host"| default "github.com" }}` in case we want to use that to generate the full repo below?

Reply:
good point! fixed in https://github.com/rancher-sandbox/cOS-toolkit/pull/493/commits/da26962f9c2eca70f1be20dc36395276ab5fa494 . let's see if CI tests are passing :)